app/lib/request.rb

```diff
@@ -57,7 +66,7 @@ class Request
   # We enforce a 5s timeout on DNS resolving, 5s timeout on socket opening
   # and 5s timeout on the TLS handshake, meaning the worst case should take
   # about 15s in total
-  TIMEOUT = { connect: 5, read: 10, write: 10 }.freeze
+  TIMEOUT = { connect_timeout: 5, read_timeout: 10, write_timeout: 10, read_deadline: 30 }.freeze
```

Comment:
The comment above doesn't seem to be accurate?

Reply:
It's accurate but misleading, it's about the connect time and it describes our socket class override.

Reply:
Ah, okay! So `connect_timeout` is the 5s for each in the comment, and the `read_timeout`, `write_timeout` and `read_deadline` are all the other values, which aren't in the comment?